modules/web-crypto-backend/test/synchronous_random_values.test.ts

```diff
@@ -4,9 +4,8 @@
 /* eslint-env mocha */
 
 import { expect } from 'chai'
+import { generateSynchronousRandomValues } from '../src/synchronous_random_values'
 import { synchronousRandomValues } from '../src/index'
```

Comment:
nitpick: can these can be loaded from the same module?

Reply:
Yes, they can, but I separate them because the one from `index` in the one that is exposed publicly.
Today, they are the same, but they do not need to be, so I lean towards doing it this way...